lib/index.js

```diff
@@ -110,3 +108,19 @@ function mergeOptions(options) {
 
   return merged;
 }
+
+/**
+ * Returns a function to be used by postcss-import
+ * Lint each imported component with postcss-bem-linter
+ *
+ * @param {Object} options
+ */
+function lintImportedFiles(options) {
+  return function (css, filename) {
+    return postcss([
+      bemLinter(options['postcss-bem-linter']),
```

Comment:
Passing it directly here, as the presence of an empty object (the usual fallback) causes `bem-linter` to forgo its defaults and this throws an error.